src/Components/Components/test/ParameterViewTest.cs

```diff
@@ -20,7 +20,7 @@ public void CanInitializeUsingComponentWithNoDescendants()
             {
                 RenderTreeFrame.ChildComponent(0, typeof(FakeComponent)).WithComponentSubtreeLength(1)
             };
-            var parameters = new ParameterView(frames, 0);
+            var parameters = new ParameterView(ParameterViewLifetime.Unbound, frames, 0);
```

Comment:
Could we add another test that verifies that `ParameterView.ToDictionary` also has this behavior? We wouldn't want the behavior to inadvertently break if we change the implementation.

Reply:
I'm not sure what aspect of `ParameterView.ToDictionary` you mean. That method's respect for lifetime bounds is tested here: https://github.com/aspnet/AspNetCore/pull/14541/files#diff-0e4da315f2f63dc5e8f97fad2558f5b4R3730

Do you think that's sufficient, or do you mean something else?

Reply:
Just in case, I'm also adding an explicit pure unit test for this in `ParameterViewTest.cs` (which partially duplicates the coverage in `RendererTest.cs`) to be super clear that it's a behavior of `ParameterView` and not something that only happens in conjunction with `Renderer`.